cypress/components/dialog/dialog.cy.tsx

```diff
@@ -404,274 +398,60 @@ context("Testing Dialog component", () => {
-    it("should pass accessibility tests for Dialog component using focusFirstElement", () => {
-      CypressMountWithProviders(<stories.DialogComponent />);
-
-      cy.checkAccessibility();
-    });
-
-    it("should pass accessibility tests for Dialog component disabling autofocus", () => {
-      CypressMountWithProviders(<stories.DialogComponent disableAutoFocus />);
-
-      cy.checkAccessibility();
-    });
-
-    it("should pass accessibility tests for Dialog component with Toast", () => {
-      CypressMountWithProviders(<stories.DialogComponentWithToast />);
-
-      buttonDataComponent()
-        .click()
-        .then(() => {
-          // eslint-disable-next-line no-unused-expressions
-          cy.checkAccessibility();
-        });
-    });
-
-    it("should pass accessibility tests for Dialog Editable story", () => {
-      CypressMountWithProviders(<defaultStories.Editable />);
-
-      openPreviewButton()
-        .click()
-        .then(() => {
-          // eslint-disable-next-line no-unused-expressions
-          cy.checkAccessibility();
-        });
-    });
-
-    it("should pass accessibility tests for Dialog WithHelp story", () => {
-      CypressMountWithProviders(<defaultStories.WithHelp />);
-
-      openPreviewButton()
-        .click()
-        .then(() => {
-          // eslint-disable-next-line no-unused-expressions
-          cy.checkAccessibility();
-        });
-    });
-
-    it("should pass accessibility tests for Dialog DynamicContent story", () => {
-      CypressMountWithProviders(<defaultStories.DynamicContent />);
-
-      openPreviewButton()
-        .click()
-        .then(() => {
-          // eslint-disable-next-line no-unused-expressions
-          cy.checkAccessibility();
-        });
-    });
-
-    it("should pass accessibility tests for Dialog FocusingADifferentFirstElement story", () => {
-      CypressMountWithProviders(
-        <defaultStories.FocusingADifferentFirstElement />
-      );
-
-      getComponent("button")
-        .eq(0)
-        .click()
-        .then(() => {
-          // eslint-disable-next-line no-unused-expressions
-          cy.checkAccessibility();
-        });
-    });
-
-    it("should pass accessibility tests for Dialog OverridingContentPadding story", () => {
-      CypressMountWithProviders(<defaultStories.OverridingContentPadding />);
-
-      openPreviewButton()
-        .click()
-        .then(() => {
-          // eslint-disable-next-line no-unused-expressions
-          cy.checkAccessibility();
-        });
-    });
-
-    it("should pass accessibility tests for Dialog OtherFocusableContainers story", () => {
-      CypressMountWithProviders(<defaultStories.OtherFocusableContainers />);
-
-      openPreviewButton()
-        .click()
-        .then(() => {
-          // eslint-disable-next-line no-unused-expressions
-          cy.checkAccessibility();
-        });
-    });
+  it("should pass accessibility tests for Dialog OverridingContentPadding story", () => {
```

Comment:
As above, I think this test case should be within the brackets of the other accessibility tests `Describe...`. It's out on its own here.